compiler/test/fail_compilation/failcstuff1.c

```diff
@@ -7,9 +7,7 @@ fail_compilation/failcstuff1.c(152): Error: `;` or `,` expected
 fail_compilation/failcstuff1.c(153): Error: `void` has no value
 fail_compilation/failcstuff1.c(153): Error: missing comma
 fail_compilation/failcstuff1.c(153): Error: `;` or `,` expected
-fail_compilation/failcstuff1.c(157): Error: expression expected, not `struct`
-fail_compilation/failcstuff1.c(157): Error: found `S22028` when expecting `)`
-fail_compilation/failcstuff1.c(157): Error: missing comma or semicolon after declaration of `test22028`, found `ident` instead
+fail_compilation/failcstuff1.c(157): Error: identifier not allowed in abstract-declarator
```

Comment:
The code that creates this error is:
```c
int test22028 = sizeof(struct S22028 ident);
```

IMO, new error is better than the old. Even if importC grammar errors can be a bit impenetrable.